### PR TITLE
Beautify names fixes #2811

### LINKED ISSUE #2811: Visualization auto naming needs to be fixed
Why not just replace underscores "_" with spaces
and then uppercase the first letter of each word

Currently it looks like this

screen shot 2015-03-17 at 7 24 24 pm

### PR DESCRIPTION
**Pull request by juanignaciosl**
@Kartones CR this fix for #2811, please.

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2756/))

File: app/models/table.rb [hunk 1]
```diff
@@ -1419,6 +1419,11 @@ def remove_organization_access
 
   private
 
+  def beautify_name(name)
+    return name unless name
+    name.gsub('_', ' ').split.map(&:capitalize).join(' ')
```
Comment (Kartones) [inline app/models/table.rb]: why not directly splitting specifying the character? `name.split('_').map(&:capitalize).join(' ')` We save the `gsub`

**Comment by Kartones** [comment]
:+1:

File: app/models/table.rb (hunk 1 above)
Comment (juanignaciosl) [inline app/models/table.rb]: Because you also need to split existing spaces in order to capitalize all words.

Comment (Kartones) [inline app/models/table.rb]: True, I forgot that it has a " Map" added

**Pull request merged**